main.rb

```diff
@@ -0,0 +1,74 @@
+require './app'
+
+class LibraryApp
+  def display_options
+    puts 'Options:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List rentals for a person'
+    puts '7. Quit'
+  end
+
+  def list_rentals(people)
+    print 'Enter person id: '
+    person_id = gets.chomp.to_i
+    list_rentals_for_person(person_id, people)
+  end
+
+  def handle_list_books(books)
+    list_all_books(books)
+  end
+
+  def handle_list_people(people)
+    list_all_people(people)
+  end
+
+  def handle_create_person(people)
+    create_person(people)
+  end
+
+  def handle_create_book(books)
+    create_book(books)
+  end
+
+  def handle_create_rental(books, people)
+    create_rental(books, people)
+  end
+
+  def handle_list_rentals(people)
+    list_rentals(people)
+  end
+
+  def handle_choice(choice, books, people)
+    case choice
+    when 1 then handle_list_books(books)
+    when 2 then handle_list_people(people)
+    when 3 then handle_create_person(people)
+    when 4 then handle_create_book(books)
+    when 5 then handle_create_rental(books, people)
+    when 6 then handle_list_rentals(people)
+    when 7 then exit
+    else
+      puts 'Invalid choice. Please try again.'
+    end
+  end
```

Comment:
- Currently, I can see the limit is 7 line rule, that's why it's giving linter issues but in general, it is 10. so in that case your app would not give this error. For now, you can disable these metrics using
```
# rubocop:disable Metrics/CyclomaticComplexity
...
code where it's giving issues.
....
# rubocop:enable Metrics/CyclomaticComplexity
```

the idea behind this limit is to make the code looks efficient and easy to test so always try to avoid unnecessary call.